util/algorithm_identifier.go

```diff
@@ -84,3 +85,26 @@ func CheckAlgorithmIDParamNotNULL(algorithmIdentifier []byte, requiredAlgoID asn
 
 	return errors.New("RSA algorithm appears correct, but didn't match byte-wise comparison")
 }
+
+func GetPublicKeyOID(c *x509.Certificate) (asn1.ObjectIdentifier, error) {
+
+	// see also lint_spki_rsa_encryption_parameter_not_null.go
```

Comment:
I'm not sure what you mean to highlight with this comment. Another case where we use `cryptobyte` to process the `RawSubjectPublicKeyInfo`? Could you add more context to this comment (or remove it)?

Reply:
removed